### PR TITLE
continue with the header and start with the selection window

### PR DESCRIPTION
**Pull request by haimyannai**

### CODE REVIEW

File: src/app/components/header/header.component.ts [hunk 1]
```diff
@@ -11,16 +12,54 @@ export class HeaderComponent {
   userName = 'User name';
 
   public companyList: string[];
+  router: any;
 
   constructor(private globalService: GlobalService) {}
 
+  changeusercompany() {
```
Comment (yuda85) [inline src/app/components/header/header.component.ts:19]: please use camel case when naming methods or variables

File: src/app/components/header/header.component.ts [hunk 2]
```diff
@@ -11,16 +12,54 @@ export class HeaderComponent {
...
+    if (language == 'english') {
+      console.log('choose write in english');
+    } else console.log('else language');
+  }
+
+  manualbtn() {
+    console.log('manual buttun clicked');
+  }
+
+  backToDefault() {
```
Comment (yuda85) [inline src/app/components/header/header.component.ts:40]: please make sure to use access modifiers on methods (private or public)

File: src/app/services/global.service.ts [hunk 3]
```diff
@@ -3,6 +3,43 @@ import { Injectable } from '@angular/core';
 import { map, of } from 'rxjs';
 import { Observable } from 'rxjs';
 
+interface UserDetails {
```
Comment (yuda85) [inline src/app/services/global.service.ts:6]: please make sure to export all the interfaces

Comment (haimyannai) [reply src/app/services/global.service.ts:6]: i would like you to show me how to do that...

**Review by RafiLevi81 — APPROVED** [review]

**Pull request merged**